src/chronometer.js

```diff
@@ -1,33 +1,50 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.startInterval = setInterval(() => {
```

Comment:
On the constructor, you already have a variable `intervalId` declared for storing the `setInterval` you should have used that one instead of creating a new one `startInterval`